powermeter/homeassistant.py

```
@@ -55,37 +55,46 @@ def get_json(self, path):
 
         try:
             response = self.session.get(url, headers=headers, timeout=10)
-            response.raise_for_status()  # Raise an exception for bad status codes
+            response.raise_for_status()
             return response.json()
         except json.JSONDecodeError as e:
             logger.error(f"Failed to decode JSON response from Home Assistant API: {e}")
             logger.error(
                 f"Response content: {response.text[:200]}..."
             )  # Log first 200 chars
             raise ValueError(f"Home Assistant API returned invalid JSON: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)